test/resource_test.rb

```diff
@@ -490,9 +490,9 @@ def test_multiple_register_with_fork
     assert_equal 0, timeouts
   end
 
-  def create_resource(*args)
+  def create_resource(*args, **kwargs)
     @resources ||= []
-    resource = Semian::Resource.new(*args)
+    resource = Semian::Resource.new(args.first, **kwargs)
```

Comment:
same here. `*args` not `args.first`.

Reply:
I prefer the `name, **kwargs` format here too